12LessonTwelve/in-class-project/index.html

```diff
@@ -7,4 +7,60 @@
 <body>
 
 </body>
+</html> -->
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Lesson 12</title>
+    <!-- Google font-->
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+
+</head>
+
+<body>
+    <section id="hero">
+        <div class="top row">
+            <div class="top-content">
+                <h1>Order Takeaway Or Delivery Food</h1>
+                <p>With both of your computers open collaborate with your partner.</p>
+            </div>
+        </div>
+    </section>
+
+    <section id="main">
+        <div class="bottom  col-lg-3 col-sm-6 col-xs-12">
+            <img src="img/icon_home_1.svg"></img>
+            <h3>Search by address</h3>
+            <p>Find all restaurants available in your zone.</p>
+        </div>
+        <div class="bottom  col-lg-3 col-sm-6 col-xs-12">
+            <img src="img/icon_home_2.svg"></img>
+            <h3>Search by address</h3>
+            <p>Find all restaurants available in your zone.</p>
+        </div>
+        <div class="bottom col-lg-3 col-sm-6 col-xs-12">
+            <img src="img/icon_home_3.svg"></img>
+            <h3>Search by address</h3>
+            <p>Find all restaurants available in your zone.</p>
+        </div>
+        <div class="bottom col-lg-3 col-sm-6 col-xs-12">
+            <img src="img/icon_home_4.svg"></img>
+            <h3>Search by address</h3>
+            <p>Find all restaurants available in your zone.</p>
+        </div>
+    </section>
+
+
+
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+</body>
+
 </html>

```

Comment:
Looks like there is a margin error on two of your boxes. Please fix for a higher grade.